src/DynamoCoreWpf/UI/GuidedTour/dynamo_guides.json

```diff
@@ -269,6 +269,12 @@
 				"TooltipPointerDirection": "top_left",
 				"Width": 480,
 				"Height": 260,
+				"ExitGuide": {
+					"Height": 273,
+					"Width": 480,
+					"Title": "PackagesGuideExitTitle",
+					"FormattedText": "PackagesGuideExitAcceptTerms"
```

Comment:
Are these two strings displayed as part of the popup @RobertGlobant20 @filipeotero if there are strings need to be localized, then lets not put them here. In general, we have been doing a good job here to only put control name, handler name in this Json. But for display content, let's not. I am not sure so just asking 😉